src/relay_sdk/workflows.py

```diff
@@ -17,7 +17,7 @@ class Workflows:
     def __init__(self, client: Session) -> None:
         self._client = client
 
-    def run(self, name: str, parameters: Mapping[str, Any] = {}) -> None:
+    def run(self, name: str, parameters: Mapping[str, Any] = {}) -> Mapping[str, Any]: # noqa E501
```

Comment:
I think it might be nice to have this be represented by an actual "model" type, which is luckily quite easy with Python [dataclasses](https://docs.python.org/3/library/dataclasses.html) when the data isn't nested.

```python
@dataclass
class Run:
    name: str
    run_number: int
    app_url: str

# ...
    def run(...):
      return Run(**r.json())
```

Also, we shouldn't ignore E501. You can split the line into multiple lines, so something like this will be fine:

```python
def run(self, name: str,
        parameters: Mapping[str, Any] = {}) -> Run:
```

Reply:
@impl I've made the changes, thanks for the comments! Waiting for approval.